.github/actions/spell-check/whitelist.txt

```diff
@@ -0,0 +1,3240 @@
+LPNEWCPLINFOA
+LPNEWCPLINFOW
+LPNMHDR
+lpntme
+LPPOINT
+LPPROC
+LPPROPSHEETPAGE
+LPPSHNOTIFY
+lprc
+LPRECT
+lpstr
+lpsz
+LPTSTR
+LPTTFONTLIST
+lpv
+LPVOID
+LPW
+LPWCH
+LPWINDOWPOS
+lpwpos
+lpwstr
+LRESULT
+lru
+lsb
+lsconfig
+lsproj
+lss
+lstatus
+lstrcmp
+LTEXT
+LTLTLTLTL
+ltype
+LUID
+lval
+LVB
+LVERTICAL
+LWA
+LWIN
+lwkmvj
+mailto
+majorly
+makeappx
+MAKEINTRESOURCE
+MAKEINTRESOURCEA
+MAKEINTRESOURCEW
+MAKELANGID
+MAKELONG
+MAKELPARAM
+MAKELRESULT
+malloc
+Manandhar
+MAPBITMAP
+maping
+MAPVIRTUALKEY
+MAPVK
+MAXDIMENSTRING
+MAXLENGTH
+MAXSHORT
+maxversiontested
+MAXWORD
+maybenull
+mbadolato
+Mbs
+MBUTTON
+MBUTTONDBLCLK
+MBUTTONDOWN
+MBUTTONUP
+Mbxy
+mdf
+MDL
+mdmerge
+MDs
+MEASUREITEM
+megamix
+Mehrain
+memallocator
+memcmp
+memcopy
+memcpy
+memcpying
+memmove
+memset
+MENUCHAR
+MENUCONTROL
+MENUDROPALIGNMENT
+MENUITEM
+MENUITEMINFO
+MENUSELECT
+Mersenne
+messageext
+metadata
+metaproj
+mfcribbon
+mgravell
+michaelniksa
+microsoft
+microsoftonline
+midl
+migrie
+mii
+MIIM
+mikegr
+mikemaccana
+milli
+mimetype
+mincore
+mindbogglingly
+mindestens
+mingw
+miniksa
+minkernel
+minmax
+minwin
+minwindef
+Mip
+mkdir
+MMBB
+mmcc
+MMCPL
+mmsystem
+MNC
+MODALFRAME
+modelproj
+MODERNCORE
+MONITORINFO
+MONITORINFOEXW
+MONITORINFOF
+Monofur
+monospaced
+monostate
+MOUSEACTIVATE
+MOUSEFIRST
+MOUSEHWHEEL
+mousemode
+MOUSEMOVE
+mousewheel
+MOVESTART
+msb
+msbuild
+mscorlib
+msctf
+msctls
+msdata
+msdn
+msft
+MSGCMDLINEF
+MSGF
+MSGFILTER
+MSGFLG
+MSGMARKMODE
+MSGSCROLLMODE
+MSGSELECTMODE
+msiexec
+MSIL
+msix
+msixbundle
+mspartners
+msrc
+msvcrt
+MSVS
+msysgit
+mui
+Mul
+multiline
+munged
+mutex
+muxes
+myapplet
+mybuildshare
+mydir
+myignite
+MYMAX
+Mypair
+Myval
+NAMELENGTH
+nameof
+namespace
+namespaced
+namestream
+Namquiseratal
+nano
+natvis
+naws
+nbsp
+Nc
+ncb
+NCCALCSIZE
+NCCREATE
+nchittest
+NCHITTEST'ed
+NCLBUTTONDOWN
+NCLBUTTONUP
+NCMBUTTONDOWN
+NCMBUTTONUP
+NCPAINT
+NCRBUTTONDOWN
+NCRBUTTONUP
+NCrunch
+NCXBUTTONDOWN
+NCXBUTTONUP
+NDEBUG
+ned
+NEL
+NEQ
+netcoreapp
+netstandard
+NEWCPLINFO
+NEWCPLINFOA
+NEWCPLINFOW
+newcursor
+Newdelete
+NEWINQUIRE
+NEWINQURE
+NEWPROCESSWINDOW
+NEWTEXTMETRIC
+NEWTEXTMETRICEX
+Newtonsoft
+NEXTLINE
+nf
+nfe
+niksa
+nlength
+Nls
+NLSMODE
+nn
+nnn
+NOACTIVATE
+NOAPPLYNOW
+NOCLIP
+NOCOLOR
+NOCONTEXTHELP
+NOCOPYBITS
+nodiscard
+NODUP
+noexcept
+noinline
+NOINTEGRALHEIGHT
+NOINTERFACE
+NOLINKINFO
+nologo
+NOMINMAX
+NOMOVE
+NONALERT
+nonbreaking
+nonclient
+NONCONST
+NONINFRINGEMENT
+NONPREROTATED
+nonspace
+NOOWNERZORDER
+NOPAINT
+noprofile
+NOREDRAW
+NOREMOVE
+NOREPOSITION
+noreturn
+NORMALDISPLAY
+NOSCRATCH
+NOSEARCH
+noselect
+NOSENDCHANGING
+NOSIZE
+NOSNAPSHOT
+NOTHOUSANDS
+nothrow
+NOTICKS
+NOTIMPL
+NOTNULL
+NOTRACK
+NOTSUPPORTED
+nouicompat
+NOUPDATE
+NOWAIT
+NOYIELD
+NOZORDER
+NPM
+npos
+Npqip
+NSTATUS
+ntapi
+ntcon
+ntcsrdll
+ntdef
+NTDEV
+ntdll
+ntifs
+ntlpcapi
+ntm
+NTMAKEENV
+ntrtl
+ntstatus
+ntsubauth
+NTSYSCALLAPI
+NTTARGETFILE
+nttree
+nturtl
+ntuser
+NTVDM
+ntverp
+ntvs
+NTWIN
+nuget
+Nullametrutrummetus
```

Comment:
What the what?

Reply:
An intentional run of Latin?
https://github.com/microsoft/terminal/blob/be77b2a2e98420db9d275d68f35b55ac58bc93c0/src/host/ut_host/CharRowBaseTests.cpp#L27

Reply:
Oh that's from a lorem ipsum generator but without the spaces.